Source/SimulationCore.py

```diff
@@ -0,0 +1,39 @@
+import Util.py
+import floorTile
+import Robot
+
+
+class Simulation:
+    def __init__(self, floor: dict, ai, robot: Robot):
+        """
+        floor: dict of (int, int):floorTile representing the floorplan
+        ai: any object with a method that matches the update(bool) -> (float, float) signature. Outputs should both be in the range [-1, 1].
+        robot: the Robot object
+        """
+        self.AI = ai
+        self.FloorPlan = floor
+        if robot.validate() and not Collision(robot.pos, robot.diameter / 2, floor):
+            self.robot = robot
+        else:
+            raise ValueError("Robot not valid, or collides with terrain")
+
+        self.ElapsedTime = 0
+        self.IsColliding = False
+        self.SimSpeed = 1
+        self.DefaultHertz = 60
```

Comment:
We should be using lowercase snake-case variable names so that we don't confuse them with classes.
I haven't been following best practices here, but I will update my code to match
[See pep 0008 for the style guide](https://peps.python.org/pep-0008/#function-and-variable-names)